public/docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,337 @@
+# Quickstart
+
+This page will take you through the basics of creating a Truffle project and migrating (deploying) a smart contract to a blockchain.
+
+## Creating a project
+
+To use most Truffle commands, you need to run them against an existing Truffle project. So the first step is to create a Truffle project.
+
+You can create a bare project template, but for those just getting started, you can use [Truffle Boxes](/boxes), which are example applications and project templates. We'll use the [MetaCoin box](/boxes/metacoin), which creates an [ERC20-style token](https://en.wikipedia.org/wiki/ERC20):
+
+1. Create a new directory for your Truffle project:
+
+   ```shell
+   mkdir MetaCoin
+   cd MetaCoin
+   ```
+
+1. Download ("unbox") the MetaCoin box:
+
+   ```shell
+   truffle unbox metacoin
+   ```
+
+   <p class="alert alert-info">
+   <strong>Note</strong>: You can use the `truffle unbox <box-name>` command to download any of the other Truffle Boxes.
+   </p>
+
+   <p class="alert alert-info">
+   <strong>Note</strong>: To create a bare Truffle project with no smart contracts included, use `truffle init`.
+   </p>
+
+Once this operation is completed, you'll now have a project structure with the following items:
+
+* `contracts/`: Directory for [Solidity contracts](./contracts)
+* `migrations/`: Directory for [scriptable deployment files](./migrations#migration-files)
+* `test/`: Directory for test files for [testing your application and contracts](./testing)
+* `truffle.js`: Truffle [configuration file](/docs/advanced/configuration)
+
+## Exploring the project
+
+<p class="alert alert-info">
+<strong>Note</strong>: This page is just a quickstart, so we're not going to go into much detail here. Please see the rest of the Truffle documentation to learn more.
+</p>
+
+1. Open the `contracts/MetaCoin.sol` file in a text editor. This is a smart contract (written in Solidity) that generates a MetaCoin token. Note that this also references another Solidity file `contracts/ConvertLib.sol` in the same directory.
```

Comment:
`generates` feels out of place here. Maybe `manages`? `creates` also feels nice, though it's closer to `generates`.

Reply:
I'd go with `creates`, unless there's some other mechanism at work (as in, I thought the contract was creating/generating a token, but maybe not?).